faunadb/errors.py

```diff
@@ -12,12 +14,35 @@ def __init__(self, description, response_data):
 
 
 #region FaunaHttpError
+
 class FaunaError(Exception):
   """
   Error returned by the FaunaDB server.
   For documentation of error types, see the `docs <https://faunadb.com/documentation#errors>`__.
   """
 
+  @staticmethod
+  def maybe_raise_from_status_code(code, response_dict):
+    # pylint: disable=no-member, too-many-return-statements
+    if 200 <= code <= 299:
```

Comment:
Don't think I like that. `200 <= code <= 299` is the general case, so it makes sense for both readability and efficiency to put it first.